devices/Adxl343/ADXL343Samples/Program.cs

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Numerics;
+using System.Threading;
+using nanoFramework.Hardware.Esp32;
+using Iot.Device.ADXL343;
+using Iot.Device.ADXL343Lib;
+
+namespace ADXL343Samples
+{
+    public class Program
+    {
+        private const bool useESP32 = true;
```

Comment:
For constants, we are using uppercase to stat with, so it will be `UseEsp32`
And for the naming convention, we use Esp instead of ESP, etc